src/sentry/integrations/slack/webhooks/event.py

```diff
@@ -39,17 +47,39 @@ class SlackEventEndpoint(SlackDMEndpoint):
     XXX(dcramer): a lot of this is copied from sentry-plugins right now, and will need refactoring
     """
 
+    _METRIC_FAILURE_KEY = SLACK_EVENT_ENDPOINT_FAILURE_DATADOG_METRIC
+    _METRICS_SUCCESS_KEY = SLACK_EVENT_ENDPOINT_SUCCESS_DATADOG_METRIC
+
     authentication_classes = ()
     permission_classes = ()
     slack_request_class = SlackEventRequest
 
     def reply(self, slack_request: SlackDMRequest, message: str) -> Response:
+        logger_params = {
+            "integration_id": slack_request.integration.id,
+            "team_id": slack_request.team_id,
+            "channel_id": slack_request.channel_id,
+            "user_id": slack_request.user_id,
+            "channel": slack_request.channel_id,
+            "message": message,
+        }
+
         payload = {"channel": slack_request.channel_id, "text": message}
-        client = SlackClient(integration_id=slack_request.integration.id)
-        try:
-            client.post("/chat.postMessage", data=payload, json=True)
-        except ApiError as e:
-            logger.error("slack.event.on-message-error", extra={"error": str(e)})
+        if options.get("slack.event-endpoint-sdk") and slack_request.integration.id in options.get(
+            "slack.event-endpoint-sdk-integration-ids"
+        ):
```

Comment:
i think this should be OR
either the feature is GA'd OR the org is part of LA.

can you add a test for just the LA part?